gprofiler/main.py

```diff
@@ -365,6 +366,7 @@ def parse_cmd_args():
         add_config_file_help=True,
         add_env_var_help=False,
         default_config_files=["/etc/gprofiler/config.ini"],
+        conflict_handler='resolve',
```

Comment:
Why? I think the default of `'error'` makes more sense

Reply:
It's so the `--no-<PROFILER>` arguments will work with the same dest arg as `--<PROFILER>-mode`

Reply:
I didn't get it?

Reply:
There's a conflict, as I add two arguments (the `--no-X` and `--X-mode`) with the same dest arg (`X_mode`). If I didn't add the conflict handler here it wouldn't work

Reply:
But we already have it this way - for example, from `nodejs` branch:
```
    python_options = parser.add_argument_group("Python")
    python_options.add_argument(
        "--python-mode",
        dest="python_mode",
        default="auto",
        choices=["auto", "pyspy", "pyperf", "none"],
        help="Select the Python profiling mode: auto (try PyPerf, resort to py-spy if it fails), pyspy (always use"
        " py-spy), pyperf (always use PyPerf, and avoid py-spy even if it fails) or none (no runtime-specific profilers"
        " for Python).",
    )
    python_options.add_argument(
        "--no-python",
        dest="python_mode",
        action="store_const",
        const="none",
        help="Do not invoke runtime-specific profilers for Python processes (legacy option for '--python-mode none')",
    )
```

both have `dest="python_mode"`.

I think the conflict handler refers to conflicts in argument *names*.

Reply:
See https://docs.python.org/3/library/optparse.html#conflicts-between-options